contrib/seeds/makeseeds.py

```diff
@@ -19,13 +19,9 @@
 
 # These are hosts that have been observed to be behaving strangely (e.g.
 # aggressively connecting to every node).
-SUSPICIOUS_HOSTS = {
-    "130.211.129.106", "178.63.107.226",
-    "83.81.130.26", "88.198.17.7", "148.251.238.178", "176.9.46.6",
-    "54.173.72.127", "54.174.10.182", "54.183.64.54", "54.194.231.211",
-    "54.66.214.167", "54.66.220.137", "54.67.33.14", "54.77.251.214",
-    "54.94.195.96", "54.94.200.247"
-}
+with open("suspicious_hosts.txt", mode="r", encoding="utf-8") as f:
```

Comment:
Pretty sure this tries to read the file in the pwd, not in this folder. Is the script required to run in this folder, so that `pwd==this folder` already holds for other reasons?

Reply:
> Pretty sure this tries to read the file in the pwd, not in this folder. Is the script required to run in this folder, so that `pwd==this folder` already holds for other reasons?

Not sure where this script is supposed to be run from.

Reply:
It's supposed to be run from the folder it is in, at least according to the instructions in the README (that I always follow): https://github.com/bitcoin/bitcoin/blob/master/contrib/seeds/README.md